native/swift/Sources/wordpress-api/Extensions.swift

```diff
@@ -32,3 +32,23 @@ extension MiddlewarePipeline {
         self.init(middlewares: middlewares)
     }
 }
+
+public extension CommentStatus {
```

Comment:
Should this conform to `ExpressibleByStringLiteral`? I wonder if that'd help with passing it around and/or instantiating new ones (for instance, for a filter)?

Reply:
Added in https://github.com/Automattic/wordpress-rs/pull/734/commits/7147ff98e7a52e7e2902b31d8e27cd813bff2080.

Reply:
This gives us `init(stringLiteral: String)` and `init(status: String)`

Do we need both? It seems to me this is an enum type represented by a String, which I _think_ is basically the exact use case for `ExpressibleByStringLiteral `?

WDYT?

Reply:
I think they are for different use cases. `ExpressibleByStringLiteral` allows you to `let status: CommentStatus = "hello"`. But if we need to convert a `status: String` to a `status: CommentStatus`, then `CommentStatus(status: status)` is much clearer than `CommentStatus(stringLiteral: status)`.

Reply:
WDYT about just `CommentStatus(status)` for the extension initializer (like you'd do `String(someInt)`)?

Reply:
Do you mean removing the labels like https://github.com/Automattic/wordpress-rs/pull/734/commits/42785042b17721c6d3bfaf5fa1c81dbeb68417c1?

Reply:
Yeah, I think that's a decent amount nicer. Sorry for all the nitpicks 😅